services/src/main/java/org/openlearn/config/OAuth2ServerConfiguration.java

```diff
@@ -76,7 +76,7 @@ public void configure(HttpSecurity http) throws Exception {
             .and()
                 .csrf()
```

Comment:
Unrelated to your changes, but why is CSRF disabled? We will want to enable it since this app is driven by adding data from users.